cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/MetricFetcherManager.java

```diff
@@ -148,7 +148,8 @@ public boolean fetchPartitionMetricSamples(long startMs,
                                              long endMs,
                                              long timeoutMs,
                                              SampleStore sampleStore) throws TimeoutException {
-    LOG.debug("Kicking off sampling for time range [{}, {}].", startMs, endMs);
+    LOG.info("Kicking off sampling for time range [{}, {}] {}ms using {} fetchers with {}ms timeout.",
```

Comment:
Should there be a space before "ms"? Also it is not clear what does the third value mean from the text itself.

Reply:
added text to indicate what the 3d value is